tensorflow/compiler/mlir/xla/transforms/map_lhlo_to_scalar_op.h

```diff
@@ -59,69 +64,71 @@ template <typename LHLO_BinaryOp>
 using ScalarIOp = typename ScalarOp<LHLO_BinaryOp>::IOp;
 
 template <typename LhloOp>
-Operation* MapLhloOpToStdScalarOp(LhloOp lhlo_op, ArrayRef<Type> result_types,
-                                  ArrayRef<Value*> block_args, OpBuilder b) {
+Value* MapLhloOpToStdScalarOp(LhloOp lhlo_op, ArrayRef<Type> result_types,
+                              ArrayRef<Value*> block_args, OpBuilder b) {
   Type element_type = block_args.front()->getType();
+  Operation* op;
   if (element_type.isa<IntegerType>()) {
-    return b.template create<ScalarIOp<LhloOp>>(lhlo_op.getLoc(), result_types,
```

Comment:
Why not `return b.template create<ScalarIOp<LhloOp>>(lhlo_op.getLoc(), result_types, block_args, mlir::None).getResult()`. I would find that easier to read compared to the extra conditional at the end. You should be able to use getResult() here, as the created operations have a single result.